isb_web/config.py

```diff
@@ -79,6 +49,10 @@ class Settings(BaseSettings):
     # This must not ever be checked in.  Set it by doing export ORCID_CLIENT_SECRET=foobar
     orcid_client_secret: str = ""
 
+    orcid_issuer: str = "https://sandbox.orcid.org"
+
+    secret_key: str = "123456"
```

Comment:
@datadavev per discussion on this morning's call we will leave this checked in.  I will give it a better name